pyaedt/modules/Material.py

```diff
@@ -648,7 +648,17 @@ def __init__(self, parent, name, props=None):
             self.physics_type = self._props["PhysicsTypes"]["set"]
         else:
             self.physics_type = ['Electromagnetic', 'Thermal', 'Structural']
-            self._props["PhysicsTypes"] = OrderedDict({"set":['Electromagnetic', 'Thermal', 'Structural']})
+            self._props["PhysicsTypes"] = OrderedDict({"set": ['Electromagnetic', 'Thermal', 'Structural']})
+        if "AttachedData" in self._props:
+            self._material_appearance = []
+            self._material_appearance.append(self._props["AttachedData"]["MatAppearanceData"]["Red"])
+            self._material_appearance.append(self._props["AttachedData"]["MatAppearanceData"]["Green"])
+            self._material_appearance.append(self._props["AttachedData"]["MatAppearanceData"]["Blue"])
+        else:
+            self._material_appearance = [128, 128, 128]
+            self._props["AttachedData"] = OrderedDict({"MatAppearanceData":
```

Comment:
Any reason why we use an ordered dictionary here?

Reply:
The entire `self._props` need to be a nested ordered dictionary (see line 564), otherwise AEDT will fail if the arguments are not given in the expected order. It's crazy I know.....

Reply:
Thanks for the explanation!